backend/app/conversations/callConvo.js

```diff
@@ -15,34 +15,46 @@ function startCallConversation(user, userConversation, representatives, campaign
   }
 
   // then begin the conversation
-  return Reps.find({ _id: { $in: representatives } }).exec().then(representatives => {
-    const convoData = {
-      firstName: user.firstName,
-      issueMessage: campaignCall.message,
-      issueLink: campaignCall.issueLink,
-      shareLink: campaignCall.shareLink,
-      issueSubject: campaignCall.title,
-      issueTask: campaignCall.task,
-      campaignCall: campaignCall.toObject({ virtuals: false }), // without toObject mongoose goes into an infinite loop on insert
-      userConversationId: userConversation._id,
-      representatives: representatives.map(representative => ({
-        repType: representative.legislatorTitle,
-        repName: representative.official_full,
-        repId: representative._id,
-        repImage: representative.image_url,
-        repPhoneNumber: representative.phone,
-        repWebsite: representative.url,
-      })),
-      currentRepresentativeIndex: 0,
-      numUserCalls: 0,  // the number of calls this user has made for this campaignCall
-    }
-    // save params as convoData
-    user.convoData = convoData
+  const repsPromise = Reps.find({ _id: { $in: representatives } }).exec()
+  const userConversationCountPromise = UserConversation.count({ user: user._id, active: true }).exec()
+  return Promise.all([repsPromise, userConversationCountPromise])
+    .then(([representatives, userConversationCount]) => {
+      const isFirstTimeCaller = userConversationCount === 0
+      // only send one representative if it's the user's first time calling
+      if (isFirstTimeCaller) {
+        representatives = representatives.slice(0, 1)
+      }
+      const convoData = {
```

Comment:
could be good to store isFirstTimeCaller in  convoData as well

(if there ever is an error, convoData also gets logged to sentry)